typescript-selenium-webdriver/azure-pipelines.yml

```diff
@@ -1,26 +1,63 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+
+# This file defines an Azure Pipelines Continuous Integration (CI) build.
```

Comment:
Is it correct to say that this define a CI build?
I don't see any condition here where we are mentioning CI (or PR for that matter).
There's also the case of going to the **Triggers** section of the build and override the YAML triggers for both CI and PR

Reply:
I picked the wording "define" (instead of, say, "configure") for consistency with the [Pipelines "Understand the `azure-pipelines.yml` file" docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/customize-pipeline?view=azure-devops#understand-the-azure-pipelinesyml-file):

> A pipeline is defined using a YAML file in your repo. Usually, this file is named `azure-pipelines.yml` and is located at the root of your repo.

The default behavior with no triggers specified and no overrides in the UI is for a pipeline to run in CI mode for every commit to any branch and also in PR mode for every PR update targeting any branch. I didn't want to go too far into trigger documentation since it seemed somewhat out of scope for this sample, but I agree maybe an intro description would be helpful for an unfamiliar new user. Updated wording to try to balance these better.